### PR TITLE
#12744867: retrieve feeds

### PR DESCRIPTION
**Pull request by bolorundurowb**
- Adds logic to retrieve and persist feeds

### CODE REVIEW

File: cron/FeedsJob.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const db = require('./../models');
+const Config = require('./../config/Config');
+const Logger = require('./../config/Logger');
+
+class FeedsJob {
+  static async run() {
+    try {
+      const response = await axios.get(Config.feedUrl);
+      const newsItems = $('.td-ss-main-content > div > div.item-details', response.data);
```
Comment (Saint-lawal) [inline cron/FeedsJob.js]: @bolorundurowb  I'm guessing what it returns is html?

File: models/Subscription.js [hunk 2]
```diff
@@ -20,6 +20,11 @@ const SubscriptionModel = mongoose.model('Subscription', new Schema({
   email: {
     type: String,
     required: true
+  },
+  frequency: {
```
Comment (Saint-lawal) [inline models/Subscription.js]: This should be updated in the controllers as well.

File: cron/FeedsJob.js (hunk 1 above)
Comment (bolorundurowb) [reply cron/FeedsJob.js]: yes it does

File: models/Subscription.js (hunk 2 above)
Comment (bolorundurowb) [reply models/Subscription.js]: Done

**Pull request merged**